packages/compartment-mapper/src/parse-cjs.js

```diff
@@ -41,7 +41,7 @@ export const parseCjs = (
    */
   const execute = (moduleEnvironmentRecord, compartment, resolvedImports) => {
     const functor = compartment.evaluate(
-      `(function (require, exports, module, __filename, __dirname) { ${source} //*/\n})\n//# sourceURL=${location}`,
+      `(function (require, exports, module, __filename, __dirname) { 'use strict'; ${source} })\n`,
```

Comment:
Moving the pragma into each individual functor and the generated linker was needed to make bundles suitable inlined anywhere an expression is accepted, like `(${source})`.